force-app/test/jest-mocks/lightning/platformShowToastEvent.js

```diff
@@ -14,5 +28,17 @@ export class ShowToastEvent extends CustomEvent {
             bubbles: true,
             detail: toast
         });
+
+        // Set exposed toast values for showToastEventCalledWith
```

Comment:
Haven't worked with classes in years; can someone double-check this?

Reply:
@tessaSAC you shouldn't test the toast this way.

Check out [this example from LWC Recipes](https://github.com/trailheadapps/lwc-recipes/blob/945632fdb4904ae3a9e17bad2fe53d85f7fe7e91/force-app/main/default/lwc/miscToastNotification/__tests__/miscToastNotification.test.js#L33). All you need to do is to register an event handler in your test and you'll be able to capture the toast data.

With this approach you can roll back to a [simple stub](https://github.com/trailheadapps/lwc-recipes/blob/945632fdb4904ae3a9e17bad2fe53d85f7fe7e91/force-app/test/jest-mocks/lightning/platformShowToastEvent.js).